app/konnect/mesh-manager/service-mesh.md

```diff
@@ -34,54 +38,196 @@ After creating the global control plane, you must add a zone to that control pla
+  ```bash
+  kubectl apply -f demo.yaml
+  ```
+
+1. Port forward the service to the namespace on port `5000`:
+  ```bash
+  kubectl port-forward svc/demo-app -n kong-mesh-system 5000:5000
+  ```
+
+1. In a browser, go to `127.0.0.1:5000`. <!--WHY?-->
+
+You can see the services the Kubernetes demo app added by navigating to **Mesh Manager** in the sidebar of {{site.konnect_short_name}}, selecting the `clothing-website` global control plane and clicking **Meshes** in the sidebar. You can view the services associated with that mesh by clicking **Default** and the **Services** tab.
 
 For more information about the Kubernetes demo app, see [Explore {{site.mesh_product_name}} with the Kubernetes demo app](/mesh/latest/quickstart/kubernetes/).
 
 ## Configure `kumactl` to connect to your global control plane
 
-`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can create, read, update, and delete resources in {{site.mesh_product_name}} in Universal/{{site.konnect_short_name}} mode.
-
-You connect `kumactl` to the global control plane in {{site.konnect_short_name}} so that you can run commands against the control plane.
+`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can create, read, update, and delete resources in {{site.mesh_product_name}} in Universal/{{site.konnect_short_name}} mode. Since we are deploying {{site.mesh_product_name}} in Kubernetes mode for this tutorial, `kumactl` is read-only. Instead, `kubectl` is used to apply changes to {{site.mesh_product_name}}. Although we don't use `kumactl` in this tutorial because we are using Kubernetes, it's still best practice to configure it to connect to your global control plane.
 
-1. From the left navigation menu in {{site.konnect_short_name}}, open {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `example-cp` control plane.
+1. From the left navigation menu in {{site.konnect_short_name}}, open {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `clothing-website` control plane.
 1. Select **Configure kumactl** from the **Global Control Plane Actions** dropdown menu and follow the steps in the wizard to connect `kumactl` to the control plane.
 1. Verify that the services you added from the previous section with the Kubernetes demo app are running correctly:
-```bash
-kumactl get dataplanes
-```
-If your data planes were configured correctly with the demo app, the output should return all four data planes. 
+  ```bash
+  kumactl get dataplanes
+  ```
+If your data planes were configured correctly with the demo app, the output should return all four data planes: `frontend`, `backend`, `postgres`, and `redis`.
 
 You can now issue commands to your global control plane using `kumactl`. You can see the [`kumactl` command reference](/mesh/latest/explore/cli/#kumactl) for more information about the commands you can use.
```

Comment:
Is there anything we'd want to add to a section about using `kumactl` like an example of any sort that would be useful in this scenario, or is that kind of pointless because we're using Kubernetes here? Why should we configure `kumactl` to connect to the global control plane at all if we're using `kubectl` for everything?